packages/harpguru-core/src/components/option/option.tsx

```diff
@@ -52,7 +65,15 @@ export const Option = (props: OptionProps): React.ReactElement => {
     >
       <View style={[styles.option, dynamicStyles.activeSwipeStyle]}>
         <OptionTitle>{title}</OptionTitle>
-        <OptionValue>{optionId}</OptionValue>
+        <Animated.View
+          style={[
+            {
+              transform: [{ scale: optionUpdateTransition }],
+            },
+          ]}
+        >
+          <OptionValue>{optionId}</OptionValue>
```

Comment:
This should be a RenderableToneTuple I think.

Reply:
This will take some thought since it will have to play nicely with the options which aren't pitch values. I've raised an issue to look at this in more detail:  https://github.com/js-jslog/harpguru/issues/13